app/features/syllabus_generator/document_loaders.py

```diff
@@ -105,167 +105,210 @@ def load(self, url):
         return documents
     
 def load_pdf_documents(pdf_url: str, verbose=False):
-    pdf_loader = FileHandler(PyPDFLoader, "pdf")
-    docs = pdf_loader.load(pdf_url)
-
-    if docs:
-        split_docs = splitter.split_documents(docs)
-
-        if verbose:
-            logger.info(f"Found PDF file")
-            logger.info(f"Splitting documents into {len(split_docs)} chunks")
+    try:
```

Comment:
I don't this try-catch is required here, Because .load() from the FileHandler class has try-catch which handles the errors.

Reply:
Yes, indeed the FileHandler class would handle those errors. Initially, I thought implementing specific try-catch blocks would provide better understanding and error handling as requested by the team. However, I agree that these try-catch blocks are redundant. I'll remove them.